libraries/AP_OSD/AP_OSD_Screen.cpp

```diff
@@ -782,9 +796,9 @@ void AP_OSD_Screen::draw_pitch_angle(uint8_t x, uint8_t y)
 void  AP_OSD_Screen::draw_flightime(uint8_t x, uint8_t y)
 {
     backend->write(x, y, false, "%c", SYM_FLY);
-    backend->write (x+1, y, false, "%3d%c%02.2d", (osd->flightime/1000)/60, SYM_COL,(osd->flightime/1000) % 60);
-    
-  
+    backend->write(x+1, y, false, "%3d%c%02.2d", (osd->flightime/1000)/60, 0x3A,(osd->flightime/1000) % 60);
```

Comment:
This hardcodes the 0x3A value, and there is a whitespace missing before "(osd->flightime/1000)"